src/Features/CSharp/Portable/Completion/CSharpCompletionService.cs

```diff
@@ -46,8 +48,8 @@ internal class CSharpCompletionService : CommonCompletionService
                 new TupleNameCompletionProvider(),
                 new DeclarationNameCompletionProvider(),
                 new InternalsVisibleToCompletionProvider(),
-                new PropertySubpatternCompletionProvider()
-            );
+                new PropertySubpatternCompletionProvider(),
+                new EmbeddedLanguageCompletionProvider(CSharpEmbeddedLanguageFeaturesProvider.Instance));
```

Comment:
@jasonmalinowski this is a critical part of the design.  We don't want to have each embedded language hooked in here.  So, isntead, we just attach the EmbeddedLanguageCompletionProvider here. Then the EmbeddedLanguageCompletionProvider finds all the individual sub-language completion providers.